utils/src/main/java/org/robolectric/util/Scheduler.java

```diff
@@ -29,7 +29,11 @@
  *       Scheduler will continue looping through posted events (including future events), advancing
  *       its clock as it goes.
  * </ul>
+ *
+ * @deprecated Scheduler APIs only function when using LooperMode.LEGACY. Switch to
+ *     LooperMode.PAUSED and use {@link ShadowLooper} APIs instead.
  */
```

Comment:
Should there be a target Robolectric version for removal?

Reply:
Realistically I don't think we can remove these any time soon. There are a lot of tests that still use LEGACY Looper mode, and thus use these Scheduler APIs.